doc/ledmon.conf.pod

```diff
@@ -42,8 +42,8 @@ See also the examples section.
 
 =head2 List of configurable options:
 
-B<BLACKLIST> - Ledmon will exclude scanning controllers listed on blacklist.
-When whitelist is also set in config file, the blacklist will be ignored.
+B<EXCLUDELIST> - Ledmon will exclude scanning controllers listed on excludelist.
+When includelist is also set in config file, the excludelist will be ignored.
```

Comment:
Personally I consider "include" as incorrect in this context. For me include means "something to add additionally', so we have some controllers supported but we can add more controllers by this list.
The meaning of whitelist was "you can use only controllers listed here".
Something like:
- onlylist
- allowedlist

Fits better to me. Any comment? I'm not a native speaker so my understating could be wrong but please give me explenation.

Reply:
I got the names from a document that I'm unable to locate, but I understand what you're saying.  I'll chance this to allowedlist.

Reply:
Please also put a note in documentation to highlight that legacy forms (blacklist and whitelist) are still valid but they are deprecated, and will be removed in future releases.

Reply:
Done